### PR TITLE
Homepage check

### PR DESCRIPTION
**Pull request by Marija1984**
Please review the files appdb.py, appdb2.py, home.html and about.html. For the homepage I have created a simple app leading to html pages, needs to be checked. Also, my appdb.py app is leading to users page, where you need to register and it is connected to a database. I don't know how those files can interact or should I implement the appdb2's code into appdb.. :)

### CODE REVIEW

File: appdb2.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from flask import Flask, render_template
+app = Flask(__name__)
+
+@app.route('/')
```
Comment (mariamrf) [inline appdb2.py:4]: This part tells you where to expect the output of the next function (`def home()`). Here, this means that you expect it at `<your_URL>/` which is, when running locally = `127.0.0.1:5000/` (with or without the slash at the end).

In the other file, there is a similar function that points to the same URL, but it doesn't only display the home page but rather does two things:
- If the request is a `POST` request (see comparison with other types [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)) then it adds a new user.
- If the request is a `GET` request, it gives you the homepage (in that other example, the `index.html` page).

Okay, good. What we want is actually something a bit more organized. We want:
- The `@app.route('/')` function to only give us the homepage (`GET`).
- Another route (maybe `@app.route('/user/new')` or any other URL you want) to `POST` a new user.
- Another route (maybe `@app.route('/users')`) to show us a list of the registered users.
(this is with the current state of the code, more details on registering etc when we get to that task. For the scope of this task, we only want to build an app with a homepage and also preserve the connection to the database that you already set up).

Can you think of a way now to combine both files into one app/file?

Read more about creating API endpoints [here](https://www.smashingmagazine.com/2018/01/understanding-using-rest-api/) (ignore the part about Authentication for now, we'll get to that in a later step).

**Review by mariamrf — CHANGES_REQUESTED** [review]
This looks good! About "making both apps work together": why? You need it to be one app, right?

Let's take a look at the files:

**Comment by Marija1984** [comment]
Hi Mariam,

sorry for not replying, I moved to Vienna, I came last night. So, the last
few days were kind of stressful, yesterday I spent 11 hrs on a trip. I will
get back to my tasks within these 2 or three days, I will get back to you
asap!

Thanks,
Marija

pet, 17. svi 2019. 10:43 Mariam Maarouf <notifications@github.com> je
napisao:

File: appdb2.py (hunk 1 above)
Comment (Marija1984) [reply appdb2.py:4]: Could you please look at appdb.py. It doesn't work as it should, but I don't know what to do next. It returns the table of users always. The page .../user/new doesn't exist..

**Comment by mariamrf** [comment]
Feel free to close this one!

**Pull request closed without merging**